distmv/normal.go

```diff
@@ -354,3 +355,31 @@ func (n *Normal) setSigma() {
 		n.sigma.FromCholesky(&n.chol)
 	})
 }
+
+// TransformNormal transforms the vector src, generated from a multidimensional
+// standard normal, to one generated under the distribution specified by the
+// receiver. If dst is non-nil, the result will be stored into dst, otherwise
+// a new slice will be allocated. TransformNormal will panic if the length of
+// src is not the dimension of the Normal, or if dst is non-nil and len(src) !=
+// len(dst).
+func (n *Normal) TransformNormal(dst, src []float64) []float64 {
```

Comment:
The doc suggests that other distributions would have a similar `Transform(Normal)?` method which takes a multidimensional standard normal and returns values from the receiver's distribution, so you might need `TransformUniform` as well.  In that case the name repetition is just a coincidence and is appropriate.

I might be reading too much into the docs though.